api/v1/utils/rsvp.js

```diff
@@ -0,0 +1,10 @@
+var _ = require('lodash');
+var ATTENDANCE_REPLY = ['YES', 'NO', 'YES_TO_CREATE'];
```

Comment:
See comment towards the end. I don't think this is how we want to structure our responses.